### PR TITLE
[DEV APPROVED] Sfgb updates

### PR DESCRIPTION
**Pull request by willhall88**
**Target Process ticket**
https://moneyadviceservice.tpondemand.com/entity/9969-updates-to-mas-footer

**Summary**
Currently this PR addresses changes to the MAS footer. Adding the SFGB logo, updating the background-colour and slightly changing some translations. There will be some styles that come through from Yeast in this PR - https://github.com/moneyadviceservice/yeast/pull/47
I've added the background colour in Yeast as it looks like this will be used in publify as well as here in frontend. Once the Yeast PR is approved there will be one more commit to this PR to update yeast to the new version

These changes need to be rolled out on 2nd January and because of the deployment freeze over the christmas period all the changes will be worked on and ideally rolled out together, tested on staging and be the first deployment on 2nd january. 

**Examples (optional)**
free_and_impartial_money_advice__set_up_by_government_-_money_advice_service_and_comparing_master___sfgb-updates_ _moneyadviceservice_frontend

free_and_impartial_money_advice__set_up_by_government_-_money_advice_service_and_free_and_impartial_money_advice__set_up_by_government_-_money_advice_service

free_and_impartial_money_advice__set_up_by_government_-_money_advice_service

free_and_impartial_money_advice__set_up_by_government_-_money_advice_service

free_and_impartial_money_advice__set_up_by_government_-_money_advice_service

free_and_impartial_money_advice__set_up_by_government_-_money_advice_service

### CODE REVIEW

File: app/views/shared/_footer_secondary.html.erb [hunk 1]
```diff
@@ -28,6 +28,17 @@
           <% end %>
         </li>
 
+        <li class="footer-secondary__list-item footer-secondary__sfgb-link">
+          <a class="footer-secondary__link" href="https://singlefinancialguidancebody.org.uk/" target="_blank">
```
Comment (willhall88) [inline app/views/shared/_footer_secondary.html.erb]: @willhall88 don't forget to add the aria-label and any hidden text

File: config/locales/cy.yml [hunk 2]
```diff
@@ -260,9 +260,12 @@ cy:
     hidden_accessibility: Ffurflen adborth yn agor mewn ffenestr newydd
 
   footer:
+    sfgb_part_of: Rhan o'r 
+    sfgb_singe_financial: Corff Sengl ar
```
Comment (rhianwatkins1) [inline config/locales/cy.yml]: Typo in the yaml stopping the second line from coming in on the Welsh.
Also, not sure if caused by this, but it also says sfgb at the start in the Welsh
screenshot 2018-12-18 at 10 31 32

Comment (rhianwatkins1) [inline config/locales/cy.yml]: Typo in the yaml stopping the second line from coming in on the Welsh.
Also, not sure if caused by this, but it also says sfgb at the start in the Welsh
screenshot 2018-12-18 at 10 31 32

**Comment by willhall88** [comment]
Thanks for pairing @rhianwatkins1

File: app/assets/stylesheets/components/common/_footer_secondary.scss [hunk 3]
```diff
@@ -103,8 +104,33 @@
...
 
+.footer-secondary__sfgb-text {
+  display: inline-block;
+  & > * {
+    display: block;
+  }
+}
+
+.footer-secondary__sfgb-link {
+  margin-top: 0.75rem;
```
Comment (davidtrussler) [inline app/assets/stylesheets/components/common/_footer_secondary.scss]: Could we write this as a factor of $baseline-unit (*2 specifically in this case)?

Comment (willhall88) [reply app/assets/stylesheets/components/common/_footer_secondary.scss]: thanks @davidtrussler - updated this now

File: app/assets/stylesheets/components/common/_footer_secondary.scss [hunk 4]
```diff
@@ -103,8 +104,33 @@
...
   margin-right: 4px;
 }
+
+.footer-secondary--inner {
+  @include column(12);
+  @include respond-to($mq-l) {
+    display: -ms-flex;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```
Comment (jiveturkeyJay) [inline app/assets/stylesheets/components/common/_footer_secondary.scss]: Probably going to need the prefixes for justify-content and align-items in ie
-ms-flex-align: center;
-ms-flex-pack: justify;

Comment (willhall88) [reply app/assets/stylesheets/components/common/_footer_secondary.scss]: @jiveturkeyJay - added these in, are you happy with the way I've ordered all these styles?

Comment (jiveturkeyJay) [reply app/assets/stylesheets/components/common/_footer_secondary.scss]: Looks good to me

**Pull request merged**